Makefile

```diff
@@ -217,7 +217,7 @@ run-unit-test:
 
 monitor-r53-connection:
 	attempt_counter=1
-	max_attempts=10
+	max_attempts=100
```

Comment:
Mmh, this could significantly increase the time for this pipeline

Reply:
Yes it could, but the timeline is set to time out anyway after 3.5 hours and the route 53 sometimes takes over 20/30 mins to come up, so we have to wait anyway